.github/workflows/integrate.yaml

```diff
@@ -17,7 +17,9 @@ jobs:
       run: sudo apt install tox
 
     - name: Lint code
-      run: tox -e lint
+      run: |
+        tox -e tensorboard-controller-lint
+        tox -e tensorboards-web-app-lint
```

Comment:
What do you think about using a matrix here rather than calling each out?  something like:
```
    strategy: 
      matrix:
        charm: [tensorboard-controller, tensorboard-web-app]
      ...
        run: tox -e ${{ matrix.charm }}-lint
```

That way which charm is failing the test is visible in the action UI